### DISCUSSION TITLE
Prodipe ASIO driver issue

### DISCUSSION THREAD

**Discussion by al1dg** [jamulussoftware/jamulus#1471] [Technical Support & Help]
Hi
On windows 10, I use a Prodipe ST-USB Lanen microphone with Prodipe ASIO 2.0 Driver and last Jamulus 3.7.0
The ASIO Driver hangs and so does Jamulus when I try to adapt the buffer size that is set by default to 512.
It seems also that Jamuls is not recocgnized by the Prodipe ASIO driver when I try to setup the parameters thorugh the offline way.
Would it be possible to see if this Prodipe Driver could be supported?
Bests
alain

**Comment by gilgongo** [comment]
Are you able to use ASIO4all? Also, I assume you can use the driver OK for other things, is that right? So it's just Jamulus that's having the problem with it?

**Comment by al1dg** [comment]
Thank you Jonathan

I'll try ASIO4all but it seems I have issues with it too. Ill get back to you with more test results.

I can use the microphone with WhatsApp windows application and record and test OK with ASIO Prodipe tools.

Best regards.

Alain

**Reply by dcorson-ticino-com** [reply]
Looking at the data sheet of your mic I see nothing that would exclude it from working with Jamulus.
The important parameter is the sampling rate 48kHz, set that in your parameters.
Maybe they have a new ASIO driver to download ?

**Comment by al1dg** [comment]
Thank you i'll check that.I did à tests on Ubuntu Studio and it works well...
-------- Message d'origine --------De : DonC ***@***.***> Date : 10/04/2021  16:01  (GMT+01:00) À : jamulussoftware/jamulus ***@***.***> Cc : al1dg ***@***.***>, Author ***@***.***> Objet : Re: [jamulussoftware/jamulus] Prodipe ASIO driver issue (#1471) 
Looking at the data sheet of your mic I see nothing that would exclude it from working with Jamulus.
The important parameter is the sampling rate 48kHz, set that in your parameters.
Maybe they have a new ASIO driver to download ?

**Comment by al1dg** [comment]
Hi

With the ASIO4ALL V2.15 Beta1 I finaly could have it working on Windows 10.

I had to activate in ASIO Parameters both my C-Media USB Microphone and the Conexant HD Audio Capture and Output.

Then in the Jamulus parametrs, I select the inputs for my USB-C microphone and the Outputs of the jack of the computers.

It works then.

Some times, I have noises (like echoes in a tunnel) but it might be because when I tested I was alone and didn't produce much noise myself.

I'll try singing with my choir and will get back to you about this.

Bests.

 

 

 

 

 

> Message du 10/04/21 16:01
> De : "DonC" 
> A : "jamulussoftware/jamulus" 
> Copie à : "al1dg" , "Author" 
> Objet : Re: [jamulussoftware/jamulus] Prodipe ASIO driver issue (#1471)
> 
>
>

> Looking at the data sheet of your mic I see nothing that would exclude it from working with Jamulus.
> The important parameter is the sampling rate 48kHz, set that in your parameters.
> Maybe they have a new ASIO driver to download ?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/jamulussoftware/jamulus/discussions/1471#discussioncomment-594129", "url": "https://github.com/jamulussoftware/jamulus/discussions/1471#discussioncomment-594129", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]
